lib/sinon/default-behaviors.js

```diff
@@ -181,6 +183,12 @@ module.exports = {
         return fake;
     },
 
+    resolvesThis: function resolvesThis(fake) {
+        fake.resolveThis = true;
+        fake.resolve = false;
+        fake.reject = false;
```

Comment:
I guess I can set `returnValueDefined` and `exception` flags to false to keep consistency. Do you think some other flags need to be turned off?